tests/libdrgn.py

```diff
@@ -203,10 +208,12 @@ def __repr__(self):
 
 
 class Lexer:
-    def __init__(self, func, str):
-        self._lexer = _drgn_lexer()
+    def __init__(self, func, str, cpp=False):
+        self._c_family_lexer = _drgn_c_family_lexer()
+        self._lexer = self._c_family_lexer.__getattribute__("lexer")
         self._func = func
         self._str = str.encode()
+        self._c_family_lexer.__setattr__("cpp", cpp)
```

Comment:
Similarly, I think this is equivalent:
```suggestion
        self._c_family_lexer.cpp = cpp
```